sections/02-introduction.qmd

```diff
@@ -50,7 +50,7 @@ of smaller projects via the Open Source Collective or directly on
 Github. Additional information about our products and company
 contributions for the past two years can be found in our ["Year In
 Review" blog
-posts](https://posit.co/blog/2023-posit-year-in-review/).
+posts](https://posit.co/blog/2024-posit-year-in-review/).
```

Comment:
This was probably a problem last time too, but it's a bit weird to mention "past two years" when the link goes to a post for one year. 

One option would be to change "two years" to "year", and "posts" -> "post".